tools/wptrunner/wptrunner/browsers/edge.py

```diff
@@ -6,6 +6,9 @@
                                           SeleniumRefTestExecutor)  # noqa: F401
 from ..executors.executoredge import EdgeDriverWdspecExecutor  # noqa: F401
 
+import subprocess
```

Comment:
Minor nit: standard Python practice would be to put this standard library import before all our local imports (so after the magic `from __future__ …` but before `from .base …`).

Reply:
Moved.